src/testsuite/pveclib_test.c

```diff
@@ -22,12 +22,18 @@
 
 #define __STDC_WANT_DEC_FP__    1
 
+#ifndef __clang__
+/* Disable for __clang__ because of bug involving <floatn.h>
+   incombination with -mcpu=power9 -mfloat128 */
```

Comment:
Comment says `floatn.h`, but you're skipping `stdlib.h`.
s/incombination/in combination/
Is there a bug report, can you add a reference to it?
all of this occurs again in many other places as well...

Reply:
Told Nemanja Ivanovic have not heard back yet on this one.